src/en/shaded-release/0.3.0.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Kyuubi Shaded Release Notes - 0.3.0
+date: 2024-03-07
+linked: true
+---
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+### Highlight
+
+- [KYUUBI-SHADED #33] Port ZOOKEEPER-1718 to support JLine2 in ZK client 3.4
+- [KYUUBI-SHADED #35] Create a light shaded HiveMetaStoreClient for getting DelegationToken
+- [KYUUBI-SHADED #36] Remove dependencyManagement in shaded zookeeper pom
+- [KYUUBI-SHADED #37] Split Thrift out from relocated hive-service-rpc
+- [KYUUBI-SHADED #39] Bump Thrift 0.16.0
+- [KYUUBI-SHADED #40] kyuubi-relocated-hive-metastore-client supports ZooKeeper 3.4
+- [KYUUBI-SHADED #41] Relocated Hive MetaStore client uses relocated Thrift
+
+### Credits
+
+Last but not least, this release would not have been possible without the following contributors:
+
+* Cheng Pan
+* Yifan Zhoud
```

Comment:
typo `Zhoud`

Reply:
Thanks for checking, fixed in 6a655aba52aff72ec40a2f8200258b76c51049a3